sklearn/grid_search.py

```diff
@@ -670,8 +670,17 @@ class GridSearchCV(BaseSearchCV):
     fit_params : dict, optional
         Parameters to pass to the fit method.
 
-    n_jobs : int, default=1
-        Number of jobs to run in parallel.
+    n_jobs: int, default: 1 :
+        The maximum number of estimators fit in parallel.
+
+            - If -1 all CPUs are used.
+
+            - If 1 is given, no parallel computing code is used at all,
+              which is useful for debugging.
+
+            - For n_jobs below -1, (n_cpus + 1 + n_jobs) are used.
+
+            - For n_jobs = -2, all CPUs but one are used.
```

Comment:
I think it's better to merge the last two points. Maybe something like :
```
For ``n_jobs`` below -1, ``(n_cpus + n_jobs + 1)`` are used. For example, with ``n_jobs = -2`` all CPUs but one are used.
```
What do you think ?